resources/scripts/api/auth/discord.ts

```diff
@@ -0,0 +1,8 @@
+import http from '@/api/http';
+import { AxiosResponse } from "axios";
```

Comment:
Do not use AxiosResponse in this way. Use the 'response' argument which is created in the `Promise`.